### ISSUE TITLE
X15 RevA2 Schematic, ETH_RESETn connection ambiguity

### ISSUE THREAD

**Issue by stevenj** [beagleboard/beagleboard-x15#3]
The schematic shows ETH_RESETn connected to GPIO3_17 on ball U3
According to datasheet U3 can only be GPIO5_17
GPIO3_17 is AF6
AF6 is shown on schematic as GPIO3_17 also, but as N/C

Which is correct?  
ETH_RESETn connected to U3 and GPIO5_17 
or
ETH_RESETn connected to AF6 and GPIO3_17

**Comment by Chester-Gillon** [comment]
Looking at the BeagleBoard-X15_RevA2.brd file the ETH_RESETn net is connected to ball U3, so the schematic appears to show the correct ball U3 but gives the wrong label GPIO3_17. Agree that from the AM5728 datasheet ball U3 is for gpio**5**_17.

I also note that the ETH_RESETn net is connected to the KSZ9031RNX devices U14 and U15 via resistors R316 and R317. The BeagleBoard-X15_RevA2.pdf schematic and BeagleBoard-X15_RevA2B_BOM.pdf both show that R316 and R317 are "DO NOT INSTALL". i.e. by default the ETH_RESETn net is not connected in a RevA2.

The RevA2 schematic shows that the RESET_N input on the KSZ9031RNX devices is connected to the ENET_PORZ net.

I think the end result is that the KSZ9031RNX devices are only reset by a power-on reset, which would also reset the AM5728. Did you have a need to reset the KSZ9031RNX devices from a GPIO pin?

**Comment by stevenj** [comment]
No, its a quality comment is all, I know its difficult keeping track of this many IO on a device and easy to get wrong. So I wanted to make a comment so the schematic could be updated to reflect reality at some future revision.
